unstructured/partition/pdf.py

```diff
@@ -464,7 +464,41 @@ def convert_pdf_to_images(
             yield image
 
 
-def add_pytesseract_bbox_to_elements(elements, bboxes, width, height):
+def _get_element_box(
+    boxes: List[str],
+    box_idx: int,
+    char_count: int,
+):
+    """Helper function to get the bounding box of an element.
+
+    Args:
+        element
+        boxes (List[str], optional): Defaults to [].
+        box_idx (int, optional): Defaults to 0.
+        char_count (int, optional): Defaults to 0.
+    """
+    min_x = float("inf")
+    min_y = float("inf")
+    max_x = 0
+    max_y = 0
+    for box in boxes[box_idx : box_idx + char_count]:  # noqa
```

Comment:
why can't we just pass in `boxes[box_idx: box_ids+char_count]`? So this function is simplified to iterate through input boxes.